compiler/qsc_parse/src/lex/cooked.rs

```diff
@@ -38,11 +38,11 @@ pub(crate) struct Token {
 pub(crate) enum Error {
     #[error("expected `{0}` to complete {1}, found {2}")]
```

Comment:
The Display for raw::TokenKind already puts ticks around Single.

```suggestion
    #[error("expected {0} to complete {1}, found {2}")]
```